source/ui/beamline/AMExtendedControlEditor.cpp

```diff
@@ -456,6 +462,20 @@ QSize AMExtendedControlEditor::sizeHint() const{
 	return newHint;
 }
 
+void AMExtendedControlEditor::onControlMinimumValueChanged()
+{
+	if (control_) {
```

Comment:
When Jessie and I spoke today it seemed that this was the best way to ensure consistent behaviour.
The control emits the min/max changed signals once when the values are first initialized. If enableLimitsMonitoring is called on a PV control it will after that emit the signals as often as it needs to, however the editors can always expect the signal once (assuming the control is not connected when passed in).

The reason for this is that the min and max values did not always seem to be correct by the time AMExtendedControlEditor's onConnectedChanged slot was being invoked.